cmd/kubelet/app/server.go

```diff
@@ -1257,16 +1259,23 @@ func parseResourceList(m map[string]string) (v1.ResourceList, error) {
 	rl := make(v1.ResourceList)
 	for k, v := range m {
 		switch v1.ResourceName(k) {
+		case nodev1.ResourceSwap:
```

Comment:
Currently, only swap can be configured on node level, and there is no agreement on whether to support swap at the pod level. So it is now in `nodev1 "k8s.io/api/node/v1"` instead of `"k8s.io/api/core/v1"`.

Reply:
Some context can be found in https://github.com/kubernetes/kubernetes/pull/105271#discussion_r867304228.